pkg/hostagent/api/client/client.go

```diff
@@ -16,6 +18,7 @@ import (
 type HostAgentClient interface {
 	HTTPClient() *http.Client
 	Info(context.Context) (*api.Info, error)
+	DriverConfig(context.Context, interface{}) (interface{}, error)
```

Comment:
Would more clear as:

    // Get current driver config
    DriveConfig(context.Context) (interface{}, error)

    // Patch runtime config and return updated config
    PatchDriveConfig(context.Context, interface{}) (interface{}, error)